app/dashboard/templates/profile_details.html

```diff
@@ -3,238 +3,491 @@
-                        </div>
-                        <div class="col-2">
-                          <strong>{% trans "Status" %}</strong>
-                        </div>
-                        <div class="col-10 col-sm-8 col-md-4">
-                          <strong>{% trans "Details" %}</strong>
-                        </div>
-                        <div class="col-4 col-md-4  d-none d-md-block">
-                          <strong>{% trans "Amount" %}</strong>
-                        </div>
-                      </div>
-                      {% for b in bounties %}
-                        <div class="row bounty-row" data-ref-url="{{b.url}}">
-                          <div class="col-2 d-none d-sm-block">
-                            {% if profile.handle == b.fulfiller_github_username %} 
-                              {% trans "Worker" %} 
-                            {% elif profile.handle == b.bounty_owner_github_username%}
-                              {% trans "Funder" %} 
-                            {% elif profile.handle == b.org_name %} 
-                              {% trans "Repo Owner" %} 
-                            {% else %}
-                              {% trans "Worker" %} 
-                            {% endif %}
-                          </div>
-                          <div class="col-2">
-                            {{b.status}}
+            {% endfor %}
+          {% endif %}
+
+          <button type="button" class="profile-bounties__btn-show-all btn btn-outline-secondary btn-block mt-1 py-2 hidden">Show all</button>
+        </div>
+      </div>
+
+      {% if activities %}
+        <div class="row">
+          <div class="col py-4">Latest Activities</div>
+        </div>
+        <div class="row">
+          <div class="container profile-bounties profile-bounties--activities">
+            {% for group in activities %}
+              <div class="row">
+                <div class="col-12">
+                  <p><small>{{group.title}}</small></p>
+                </div>
+                {% if group.completed %}
+                  <div class="container">
+                    <div class="row profile-bounties__activities-group mb-3 p-2">
+                      <div class="col-12">
+                        <div class="profile-bounties__group-title profile-bounties__group-title--completed">Completed {{group.completed|length}} {% if group.completed|length > 1 %}Bounties{% else %}Bounty{% endif %} <img src="/static/v2/images/check-green.svg"></div>
+
+                        {% for bounty in group.completed %}
+                          <div class="row result bounty_row {{bounty.network}}" {% if bounty.github_url %}href="{{bounty.github_url}}"{% endif %}>
+                            <div class="avatar-container col-1 justify-content-center special_tag hide_min_viewport">
+                              <img class="avatar" src="/funding/avatar?repo=https://github.com/{{bounty.bounty_owner_github_username}}"/>
```

Comment:
Can we switch this to use `{% url "" %}` for internal URL references throughout this file?